services/ratios/controllers_test.go

```diff
@@ -452,3 +459,163 @@ func (suite *ControllersTestSuite) TestGetCoinMarketsHandler() {
 	suite.Require().Greater(maxAge, 0, "Invalid max-age parameter in Cache-Control header")
 	suite.Require().LessOrEqual(maxAge, 3600, "Invalid max-age parameter in Cache-Control header")
 }
+
+func (suite *ControllersTestSuite) TestCreateStripeOnrampSessionsHandler() {
+	handler := ratios.CreateStripeOnrampSessionsHandler(suite.service)
+	// Missing payload results in 400
+	{
+		req, err := http.NewRequest("POST", "/v2/stripe/onramp_sessions", nil)
```

Comment:
nit: Is the endpoint here supposed to be `v2`? [In the docs](https://stripe.com/docs/crypto/using-the-api#create-session) I'm seeing this as `v1`.

Reply:
Sounds good, I'll resolve this one then.